chroma-common/blockdevices/blockdevice_zfs.py

```diff
@@ -241,33 +241,39 @@ def filesystem_info(self):
 
         :return: message describing zfs type and datasets found, None otherwise
         """
-        self._assert_zpool('filesystem_info')
+        try:
+            self._assert_zpool('filesystem_info')
+        except NotZpoolException:
+            blockdevice = BlockDevice(self._supported_device_types[0], self._device_path.split('/')[0])
 
-        with ZfsDevice(self._device_path, False):
-            try:
-                device_names = shell.Shell.try_run(['zfs', 'list', '-H', '-o', 'name', '-r', self._device_path]).split('\n')
+            return blockdevice.filesystem_info
+        else:
+            with ZfsDevice(self._device_path, False):
+                try:
+                    device_names = shell.Shell.try_run(['zfs', 'list', '-H', '-o', 'name', '-r', self._device_path]).split('\n')
 
-                datasets = [line.split('/', 1)[1] for line in device_names if '/' in line]
+                    datasets = [line.split('/', 1)[1] for line in device_names if '/' in line]
 
-                if datasets:
-                    return "Dataset%s '%s' found on zpool '%s'" % ('s' if (len(datasets) > 1) else '',
-                                                                   ','.join(datasets),
-                                                                   self._device_path)
-                return None
-            except OSError:                             # zfs not found
-                return "Unable to execute commands, check zfs is installed."
-            except shell.Shell.CommandExecutionError as e:    # no zpool 'self._device_path' found
-                return str(e)
+                    if datasets:
+                        return "Dataset%s '%s' found on zpool '%s'" % ('s' if (len(datasets) > 1) else '',
+                                                                       ','.join(datasets),
+                                                                       self._device_path)
+                    return None
+                except OSError:                             # zfs not found
+                    return "Unable to execute commands, check zfs is installed."
+                except shell.Shell.CommandExecutionError as e:    # no zpool 'self._device_path' found
+                    return str(e)
```

Comment:
I think the `Mount type: ...` is what we are looking for.

I'd tend to trust that output more than looking for bits and pieces as we might know them since that output is coming directly from the same library that is being used to determine the target format type and making the zfs/ldiskfs OSD plugin code selection.  It's exactly that format type determination that appears be at the root of this error message we are getting from `mount` so seeing what it thinks is on the target is the important part.

And!  If we determine that `tunefs.lustre` is telling us something different than what we expect, the test should fail right there.  In fact it would be better to pause there with a `_fetch_help()` so that we can examine it in situ.

But regardless of what we determine regarding what `tunefs.lustre` says, we should emit the output from it into the 
test output anyway for post-mortem analysis in the case where we don't detect a bad format but we hit this error anyway.